consensus/src/round_manager.rs

```diff
@@ -588,7 +588,7 @@ impl RoundManager {
         }
         self.sync_up(sync_info, author).await?;
         ensure!(
-            message_round == self.round_state.current_round(),
+            self.round_state.current_round() >= message_round,
```

Comment:
not sure why this is needed, if the message round is below the current round, it returns Ok(false) above. it's only possible that the sync_up syncs beyond the message round, which seems what should be fixed